src/server/api/routes/categories.router.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const categoriesController = require('../controllers/categories.controller');
+
+/**
+ * @swagger
+ * /categories:
+ *  get:
+ *    tags:
+ *    - Categories
+ *    summary: Get all categories
+ *    description:
+ *      Will return all categories.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  categoriesController
+    .getCategories()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /categories/{ID}:
+ *  get:
+ *    tags:
+ *    - Categories
+ *    summary: Get category by ID
+ *    description:
+ *      Will return single category with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the category to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/:id', (req, res, next) => {
+  categoriesController
+    .getCategoriesById(req.params.id)
```

Comment:
Hi @zmzade . 
```suggestion
    .getCategoryById(req.params.id)
```
That naming is wrong. That why id part is not working. You need to fix that

Reply:
done